packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -68,7 +68,13 @@ export const TerminalConfigSchema: PreferenceSchema = {
         'terminal.integrated.scrollback': {
             description: 'Controls the maximum amount of lines the terminal keeps in its buffer.',
             type: 'number',
-            default: 1000,
+            default: 1000
+        },
+        'terminal.integrated.rendererType': {
+            description: 'Controls how the terminal is rendered.',
+            type: 'string',
+            enum: ['canvas', 'dom'],
+            default: 'canvas'
```

Comment:
Do you believe a CQ is required for this new preference?
I understand VS Code has the same preference except that their default value is `auto` in which VS Code
guesses which `renderType` is the best to use.

Reply:
No, we don't need a CQ here.

Reply:
I don't think so, but maybe @benoitf may clarify.